privacyidea/lib/crypto.py

```diff
@@ -91,13 +90,12 @@ def safe_compare(a, b):
 if not PY2:
     long = int
 
-ROUNDS = 9
```

Comment:
To which number of rounds does it change, if the default is removed!?

Is this change sensible?

The documentation (https://privacyidea.readthedocs.io/en/latest/faq/crypto-considerations.html?highlight=argon2#pin-hashing) also needs adaption if the number of rounds changes!

Reply:
It would change to 2 rounds. Was 9 a suggestion for the hardening?

Reply:
We could also make the additional parameters configurable via pi.cfg with `kwargs`.

Reply:
I can not find this det